tools/babel-config.js

```diff
@@ -0,0 +1,21 @@
+// This is the same set of transformations that we will enable in a future
+// official React package
+module.exports = {
```

Comment:
That's a totally fair point. Until we have a good solution for ensuring compatibility between the compiler and the runtime helpers, I'm persuaded it's too risky/annoying to support our own subset of the runtime.